src/sweeper/sweepers/duplicates.py

```diff
@@ -1,65 +1,91 @@
 #!/usr/bin/env python
 # * coding: utf8 *
-'''
-duplicates.py
-A module that removes duplicate geometry or attributes or both
-'''
-
-import arcpy
 import re
-import os
+
 from xxhash import xxh64
 
-class DuplicateTest(object):
+import arcpy
+
+
+class DuplicateTest():
+    '''A class that finds and removes duplicate geometries or attributes or both
+    '''
     def __init__(self, workspace, table_name):
         self.report = {}
         self.workspace = workspace
         self.table_name = table_name
 
 
     def sweep(self):
+        '''A method that finds duplicate records and returns a report dictionary
+        '''
+        digests = set([])
+
+        truncate_shape_precision = re.compile(r'(\d+\.\d{2})(\d+)')
+
+        with arcpy.EnvManager(workspace=self.workspace):
+            description = arcpy.da.Describe(self.table_name)
+
+            skip_fields = ['guid', description.shapeFieldName]
```

Comment:
I think that excluding by field type might be safer than by field name for `guid` since that field name might be different.

Reply:
I’m not sure this is the safest idea. I think guid as a primary key isn’t common enough with our data to leave it. We could create a cli flag to input ignore fields if our defaults are poorly chosen. Thoughts?